src/vscode-bicep/src/commands/findOrCreateActiveBicepFile.ts

```diff
@@ -162,7 +162,7 @@ async function queryCreateBicepFile(
   return uri;
 }
 
-function compareStrings(a: string, b: string): number {
+export function compareStrings(a: string, b: string): number {
```

Comment:
Missed this last time, but this can be replaced with [String.localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare).

Reply:
Updated here:
https://github.com/Azure/bicep/commit/93b9ed61a50068e6eefe0c0b345188aabf388e37

Thanks!

Reply:
@shenglol, I had an offline conversation with @StephenWeatherford regarding this.

Looks like the locale comparison might be problematic for languages like German. Stephen mentioned some characters carry same value even though the path might be different. I am reverting the change to what it was before:
https://github.com/Azure/bicep/commit/40c1291a8181d4f8bf535f6f224ab30e29a5caa4

Reply:
String.localeCompare perhaps does the non-locale version without any params?  Not sure, but the current works.

Reply:
I thought so too, but we discussed machine settings getting picked up. I'll leave it as is for now. Let me know if I should revert it back.